widgetastic_manageiq/__init__.py

```diff
@@ -1356,11 +1356,12 @@ def find_row_on_pages(self, table, *args, **kwargs):
                                          .format(kwargs, table))
 
 
-def PaginationPane():  # noqa
+def PaginationPane(*args, **kwargs):  # noqa
```

Comment:
we don't use PaginationPane directly. It is always included into some view. 
VersionPick will be correctly resolved by widgetastic. So, it will become widget and will get parent using ordinary way.

You change makes sense only in one case when we want to use PaginationPage directly from ipython. I'll merge your changes, If you think we may have such use case in future.